apps/condo/domains/ticket/schema/TicketReportService.js

```diff
@@ -95,6 +127,93 @@ const TicketReportService = new GQLCustomSchema('TicketReportService', {
                 return { data }
             },
         },
+        {
+            access: access.canReadTicketReportAnalyticsData,
+            schema: 'getTicketReportAnalyticsData(data: TicketReportAnalyticsInput!): TicketReportAnalyticsOutput',
+            resolver: async (parent, args, context, info, extra) => {
+                const { dateFrom, dateTo, groupBy, userOrganizationId, ticketType, viewMode, addressList } = args.data
+                const isLineChart = viewMode === 'line'
+                const statuses = await getOrganizationStatuses(context, userOrganizationId)
+                const statusesMap = Object.fromEntries(statuses.map(({ type, name }) => ([type, name])))
+                const userProperties = await getOrganizationProperties(context, userOrganizationId)
+                const userPropertiesMap = Object.fromEntries(userProperties.map(({ id, address }) => ([id, address])))
+                const daysCount = moment(dateTo).diff(moment(dateFrom), 'days') + 1
+                const daysMap = Array.from({ length: daysCount }, (_, day) => moment(dateFrom).add(day, 'days'))
+                const labels = isLineChart ? statusesMap : userPropertiesMap
```

Comment:
try to use complex solution for situation when another chart will be added

Reply:
@Dimitreee I was going to use another query to collect data for the another charts. I think there is no way to find complex solution for all next charts because each report has its own grouping and also depends on different models. Could you explain this more detail?